hypershift-operator/controllers/nodepool/nto.go

```diff
@@ -70,13 +72,41 @@ func (r *NodePoolReconciler) reconcileMirroredConfigs(ctx context.Context, logr
 	// delete the redundant configs that are no longer part of the nodepool spec
 	for i := 0; i < len(existingConfigsList.Items); i++ {
 		existingConfig := &existingConfigsList.Items[i]
-		if toDelete.Has(existingConfig.Name) {
-			_, err := supportutil.DeleteIfNeeded(ctx, r.Client, existingConfig)
-			if err != nil {
-				return fmt.Errorf("failed to delete ConfigMap %s/%s: %w", existingConfig.Namespace, existingConfig.Name, err)
-			}
+		if !toDelete.Has(existingConfig.Name) {
+			continue
+		}
+		_, err := supportutil.DeleteIfNeeded(ctx, r.Client, existingConfig)
+		if err != nil {
+			return fmt.Errorf("failed to delete ConfigMap %s/%s: %w", existingConfig.Namespace, existingConfig.Name, err)
 		}
 	}
+	// NTO also generates config in the HCP namespace.
+	ntoGeneratedKubeletConfigs := &corev1.ConfigMapList{}
+	if err := r.List(ctx, ntoGeneratedKubeletConfigs, &client.ListOptions{
+		Namespace: controlPlaneNamespace,
+		LabelSelector: labels.SelectorFromValidatedSet(labels.Set{
+			nodeTuningGeneratedConfigLabel: "true",
```

Comment:
all the filters that now require "true" as value will break existing configs not setting it

Reply:
NTO  generates the its config with this label:
https://github.com/openshift/cluster-node-tuning-operator/blob/1685a8fc13765c6b0e03bb0b45a92e97fc64dbe5/pkg/performanceprofile/controller/performanceprofile/hypershift/components/handler.go#L158